pkg/asset/installconfig/nutanix/validation.go

```diff
@@ -0,0 +1,61 @@
+package nutanix
+
+import (
+	"context"
+	"github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+
+	"github.com/openshift/installer/pkg/types"
+	nutanixtypes "github.com/openshift/installer/pkg/types/nutanix"
+	"github.com/openshift/installer/pkg/types/nutanix/validation"
+)
+
+// Validate executes platform-specific validation.
+func Validate(ic *types.InstallConfig) error {
+	if ic.Platform.Nutanix == nil {
+		return field.Required(field.NewPath("platform", "nutanix"), "nutanix validation requires a nutanix platform configuration")
+	}
+
+	p := ic.Platform.Nutanix
+	if errs := validation.ValidatePlatform(p, field.NewPath("platform").Child("nutanix")); len(errs) != 0 {
```

Comment:
This validation is redundant with the validation done on the install config.
https://github.com/openshift/installer/blob/3644da52c15c6ebbf211252f7b1cfdfce3da4456/pkg/types/validation/installconfig.go#L514-L518
The validation that we want to be doing here is validation that requires a connection to the Prism Central.

Reply:
Fair point.